api.js

```diff
@@ -19,7 +19,7 @@ function Api(files, options) {
 	EventEmitter.call(this);
 
 	objectAssign(this, options);
-	this.options = options;
+	this.options = options || {};
```

Comment:
I already fixed this in https://github.com/sindresorhus/ava/commit/3fdf44d1acbe8c6f14eb3865352ce49812c4a956. Can you rebase to fix the merge conflict?